### PR TITLE
bpo-40275: Use new test.support helper submodules in tests

### PR DESCRIPTION
**Pull request by shihai1991**
Use test helpers in the following test cases:
* test_import
* test_importlib
* test_inspect
* test_site
* test_fixcid
* test_i18n
* test_lll
* test_pathfix
* test_pindent
* test_sundry

https://bugs.python.org/issue40275

### CODE REVIEW

**Comment by shihai1991** [comment]
cc @vstinner

File: Lib/test/test_import/__init__.py [hunk 1]
```diff
@@ -27,6 +27,10 @@
     TESTFN, rmtree, temp_umask, TESTFN_UNENCODABLE, temp_dir)
 from test.support import script_helper
 from test.support import threading_helper
+from test.support.import_helper import (forget, make_legacy_pyc, unlink,
+                                        unload, DirsOnSysPath)
+from test.support.os_helper import (TESTFN, rmtree, temp_umask,
+                                    TESTFN_UNENCODABLE, temp_dir)
```
Comment (vstinner) [inline Lib/test/test_import/__init__.py:33]: It seems like these changes are already merged into master via the GH-21412: commit 96a6a6d42be272a27562d98549bbffc0d1854669. Maybe rebase your PR?

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).



## CLA Missing

Our records indicate the following people have not signed the CLA:

@Akuli

For legal reasons we need all the people listed to sign the CLA before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Lib/test/test_import/__init__.py (hunk 1 above)
Comment (shihai1991) [reply Lib/test/test_import/__init__.py:33]: Oh, sorry for this terrible rebase operation, I use a new PR to replace it.

**Pull request closed without merging**